### DISCUSSION TITLE
Error during server startup [win error 126]

### DISCUSSION THREAD

**Discussion by JuanLoliR** [Acly/krita-ai-diffusion#967] [Q&A]
Hello, I would like help with this error that I get when installing the Krita -ai diffusion plugin, it is the first installation, I have an Nvidia GeForce GT 1050Ti graphics card, Windows 11, I installed Visual C++ Redistributable. but I keep getting the error.
![Error Krita](https://github.com/user-attachments/assets/1159b9f6-f0f6-4cdb-b87a-d2ad874e2305)I hope for your help, thank you

**Comment by Acly** [comment]
Please check the issues about this: #965 and #966
